contracts/protocol/v1/traders/P1Liquidation.sol

```diff
@@ -107,8 +107,8 @@ contract P1Liquidation is
         );
 
         require(
-            P1Getters(perpetual).hasAccountPermissions(taker, sender),
-            "sender does not have permissions for the taker (i.e. liquidator)"
+            P1Getters(perpetual).getIsGlobalOperator(sender),
```

Comment:
what will this change do? Didn't we want to still allow liquidators to bypass the proxy for the time being?

Reply:
Yeah we still have that one running in production but we need the next version to be only usable by the proxy